woocommerce/woocommerce-android
# Feature/8260 onboarding full screen

Part of: #8260 


### Description
This refactors the code to enable opening onboarding list in full screen mode. It also handles the case of displaying a single item in the list when there's only one item left to be completed. 

### Testing instructions

1. Create an Atomic site with pending onboarding tasks.
2. Click on "View all"  and navigate to full screen.
3. Navigate back from full screen to My store and check the scroll position is the same as before clikcing on "View all".
4. Complete all tasks except one and check that only pending task is displayed.


### Images/gif

https://user-images.githubusercontent.com/2663464/220900160-082672fb-9d51-4d5c-86d4-10d157e21a6e.mp4
